frontend/ui/widget/filechooser.lua

```diff
@@ -492,7 +496,15 @@ function FileChooser:getNextFile(curr_file)
     return next_file
 end
 
-function FileChooser:showSetProviderButtons(file, filemanager_instance, one_time_providers)
+function FileChooser:selectAllFilesInFolder()
+    for _, item in pairs(self.item_table) do
+        if item.is_file then
+            self.filemanager.selected_files[item.path] = true
+        end
+    end
+end
```

Comment:
Does this include only visible files, or all files in a folder (when there are 2 pages)? If it's the first one I'd suggest a name change to something like `selectAllFilesInView`/`selectAllVisibleFiles`, or `selectAllVisibleItems` if it selects folders too. If the second, I'd like an explicit comment

Reply:
Folders are not selected for group operations.
`Select all files in folder` selects **all** files in a folder, visible on **all** pages in file manager, based on the current view options (hidden, unsupported).

Reply:
I meant it more as in code comment ;)
Not sure if it's obvious, but I don't think it would hurt